src/configuration.ts

```diff
@@ -22,7 +22,8 @@ const isBoolean = is.boolean;
 const isString = is.string;
 const isNumber = is.number;
 
-import {Logger} from '@google-cloud/common';
+import * as types from './types';
+const Logger: types.Logger = require('@google-cloud/common').Logger;
```

Comment:
Is this the only way? Doesn't something like this work?

```ts
/// <reference path="./types" />
import {Logger} from '@google-cloud/common';
```

Reply:
I couldn't get other ways to work but I didn't try `/// ... ` because I think the style guide suggests not to use them.